packages/azure-themes/src/azure/styles/DefaultButton.styles.ts

```diff
@@ -68,29 +54,22 @@ export const DefaultButtonStyles = (theme: ITheme): Partial<IButtonStyles> => {
       fill: semanticColors.buttonTextHovered,
       border: `${StyleConstants.borderWidth} solid ${semanticColors.inputBorder}`,
     },
-    rootHovered: {
-      backgroundColor: semanticColors.buttonBackgroundHovered,
-      color: semanticColors.buttonTextHovered,
-    },
-    rootPressed: {
-      backgroundColor: semanticColors.buttonBackgroundPressed,
-      color: semanticColors.buttonTextHovered,
-    },
```

Comment:
Everything looks good except this should be added back in. Removing rootPressed results in the pressed button to look like: 

![image](https://user-images.githubusercontent.com/30805892/127398291-f73f0e77-a458-4b26-8d3f-6caf3e4e2781.png)

Reply:
Instead of overwriting rootPressed, correct semantic color buttonTextChecked instead.
![image](https://user-images.githubusercontent.com/67673432/127422380-897367ad-291a-4a3c-a24b-06f99c3bd3d4.png)